### PR TITLE
Update some of the information about the Mendix Client for the React client

### PR DESCRIPTION
**Pull request by WimJongeneel**

### CODE REVIEW

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
@WimJongeneel 
I think we should have a warning at the top of the page that the React Client is in beta.

I also think the introduction should mention the two clients with a link to how to choose between React and Dojo,

Can you write that, or would you like me to draft something?

**Comment by WimJongeneel** [comment]
@MarkvanMents I've added an info box at the top of the document

File: content/en/docs/refguide/runtime/mendix-client.md [hunk 1]
```diff
@@ -13,12 +13,20 @@ The Mendix Client runs on the end-user's device and handles the interface betwee
 
 This description of the Mendix Client is based on using the Runtime Server of an app running in the cloud. You can also run Mendix locally for testing, but this is conceptually the same.
 
+{{% alert color="info" %}}
+In Studio Pro versions 10.6.0 and above, there is an alternative version of the Mendix Client written in React. This is currently a [beta](/releasenotes/beta-features/). You can enable this new client in [App Settings](/refguide/app-settings/#react-client).
+
+The React client replaces [Dojo](https://dojotoolkit.org/) with [React](https://react.dev/) for the view layer. Other components of the Mendix Client are the same in both the React and Dojo client. See our [Release Blog](https://www.mendix.com/blog/mendix-release-10-7-go-mac-it/#react-client-public-beta) for more information.
```
Comment (MarkvanMents) [inline content/en/docs/refguide/runtime/mendix-client.md]: I think we should warn people here that the Dojo widgets won't work. Perhaps:
```suggestion
The React client replaces [Dojo](https://dojotoolkit.org/) with [React](https://react.dev/) for the view layer. This means that widgets based on Dojo will no longer work. You will get consistency errors if your app contains Dojo widgets, or you can choose **Migration mode** which will allow you to build your app but will replace incompatible widgets with a placeholder.

Other components of the Mendix Client are the same in both the React and Dojo client. See our [Release Blog](https://www.mendix.com/blog/mendix-release-10-7-go-mac-it/#react-client-public-beta) for more information.
```

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi @WimJongeneel 
Thanks for making the updates.
I have one further suggestion to clarify what happens if you swap clients but still have some Dojo widgets in your app.
Please can you have a look and see if you agree.
Other than that, this is ready to be published.

**Review by MarkvanMents — APPROVED** [review]
Looks good now. Publishing. Thanks for the PR.

**Pull request merged**